airbyte-db/lib/src/main/java/io/airbyte/db/instance/FlywayMigrationDatabase.java

```diff
@@ -91,6 +100,14 @@ private void createInternalConnection() throws Exception {
   public void close() {
     JDBCUtils.safeClose(connection);
     connection = null;
+    dslContext.close();
+    if (dataSource instanceof Closeable closeable) {
```

Comment:
I didn't know you could assign a variable of that type in this statement like this, thats cool!